linux/helper_ubuntu.sh

```diff
@@ -0,0 +1,33 @@
+# Adds the NR repo
+add_repo() {
+    apt update && apt -y install wget gnupg
+    if [ "$STAGING_REPO" = "true" ]; then
+        repo="http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/apt";
+    else
+        repo="https://download.newrelic.com/infrastructure_agent/linux/apt";
+    fi
+
+    echo "deb [arch=amd64] $repo focal main" > /etc/apt/sources.list.d/newrelic-infra.list
+
+    wget https://download.newrelic.com/infrastructure_agent/gpg/newrelic-infra.gpg -O - | apt-key add -
+    apt update
+}
+
+# Installs the agent package without the systemd dependency
+# This is needed because systemd is not available in the docker images, so the post-install script woudl fail
```

Comment:
s/woudl/would

Reply:
Didn't you heard? Oxford has deprecated `would`, the new recommended standard is `woudl`. Shame on you.

/s